#!/usr/bin/env python

PACKAGE = 'action_softbus'

from dynamic_reconfigure.parameter_generator_catkin import ParameterGenerator, str_t, double_t, bool_t

gen = ParameterGenerator()

gen.add("action_plugin", str_t, 0, "The name of the plugin for the action to use with action_softbus.", "action_softbus/DemoAction")
gen.add("action_frequency", double_t, 0, "The rate in Hz at which to run the action loop.", 20, 0, 100)

gen.add("restore_defaults", bool_t, 0, "Restore to the original configuration", False)
exit(gen.generate(PACKAGE, "action_softbus_node", "ActionSoftbus"))
